src/blocks/Contact.astro

```diff
@@ -12,13 +13,17 @@ import Tagline from 'fulldev-ui/components/Tagline.astro'
 import Text from 'fulldev-ui/components/Text.astro'
 import Textarea from 'fulldev-ui/components/Textarea.astro'
 
+const {
```

Comment:
Remove this and make it such a way that everything comes from the props, also the channels

Reply:
That would mean that we have to define the same channels in both footer and contact instead of in base though, is that intentional?